### PR TITLE
Basic document methods

### PR DESCRIPTION
**Pull request by scull7**
- ensure that inserting a document with an auto-generated id works.
- ensure that retrieving a document by identifier works.

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -31,3 +32,44 @@ test('db::info should return information about the test that was ' +
...
+
+  .then((res) => t.is(res.body.id, 'my-foo'))
+
+  .then(() => couchdb.get(DB_NAME, 'my-foo'))
+
+  .then((res) => res.body)
+
+  .then((doc) => {
+
+    t.is(doc.boo, 'bah')
```
Comment (jxbadam) [inline test/index.spec.js]: Shouldn't this be:

```
t.is(doc.boo, 'ghost')
```

File: test/index.spec.js [hunk 2]
```diff
@@ -31,3 +32,44 @@ test('db::info should return information about the test that was ' +
...
+  })
+
+)
+
+
+test('::create should store a document in the database with your ' +
+'provided identifier and then ::get should be able to retrieve your ' +
+'document by that identifier', t =>
+
+  couchdb.create(DB_NAME, 'my-foo', { boo: 'bah', moo: 'cow' })
```
Comment (jxbadam) [inline test/index.spec.js]: So fix it here too.

**Review by jxbadam — APPROVED** [review]
:shipit: :fire:

**Review by Bellk17 — APPROVED** [review]
:shipit:

File: test/index.spec.js [hunk 3]
```diff
@@ -31,3 +32,44 @@ test('db::info should return information about the test that was ' +
...
+  .then((res) => t.is(res.body.id, 'my-foo'))
+
+  .then(() => couchdb.get(DB_NAME, 'my-foo'))
+
+  .then((res) => res.body)
+
+  .then((doc) => {
+
+    t.is(doc.boo, 'Casper')
+    t.is(doc.moo, 'cow')
```
Comment (jxbadam) [inline test/index.spec.js]: If we are using a specific ghost we need to use a specific cow. `Betsy`

**Pull request merged**